script/workLoggingSystem.js

```diff
@@ -136,9 +136,96 @@ $("#submitNewTask").click(function() {
     console.log(status);
     console.log(category);
 
-    createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category);
+    //editing
+    if(taskpriority(priority)  && taskdifficulty(difficulty)  && tasktime(estimatedTime)  && taskstatus(status)  && taskcategory(category))
```

Comment:
Let's have just one method for validation of all rather then having multiple methods